### PR TITLE
feat(label): add 'label option for new flightresult component' option

### PR DESCRIPTION
**Pull request by aesthetiques**
# Alaska Airlines Pull Request

Adds the label option for auro-badge
image

This will be used in conjunction with other components to build a single result on the flight results page. No change to other functionality.

## Type of change:

Please delete options that are not relevant.

- [x] New capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by gusnaughton** [comment]
@erikalanghauser please leave your review here for http://wakeful-grape.surge.sh/

**Review by blackfalcon — APPROVED** [review]

File: src/style.scss [hunk 1]
```diff
@@ -136,11 +136,12 @@ svg {
 :host([label]) {
   background-color: transparent;
   border-color: var(--auro-color-border-primary-on-light);
-  color: var(--auro-color-text-secondary-on-light);	
-  padding: 0.375rem 0.5625rem 0.1875rem 0.5625rem;
+  color: var(--auro-color-text-secondary-on-light);
+  font-size: var(--auro-text-body-size-xs);
+  padding: 4px 6px;
```
Comment (blackfalcon) [inline src/style.scss]: Why is this moved to hard pixels? The badge supports the `fixed` feature. 

Please move back to the REM values.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please see comments about pixel values.

**Review by blackfalcon — APPROVED** [review]

**Pull request merged**